docs(tutorial/ja): add T&O at top and add remove mesh with blendshape (#479)
anatawa12/AvatarOptimizer: .docs/content/docs/tutorial/basic-usage/index.md

Basic Usage
===

Use Automatic Optimization {#trace-and-optimize}
---

There are several optimization which can perform automatically. for Avatars

- Removing Unused BlendShapes(Shape Keys)[^blend-shape]

>   - For BlendShapes with non-zero weight, there are processing load so freezing BlendShape will reduce processing load.

Author: なしに揃えたいかな

>   - Even if the weight is zero, removing the BlendShapes will reduce size of avatars.

Reviewer: ```suggestion
  - Even if the weight is zero, removing BlendShapes will reduce the size of avatars.
```

- Removing unused Behaviours such as PhysBones

>   - If a PhysBone that does not need to be swayed is enabled, for example a PhysBone where the mesh that exists as the target of the shaking is always disabled, an extra computational load is incurred.

Author: There be ~の形なら複数のほうがいいと思ってます。

>   - If a PhysBone that does not need to be swayed is enabled, for example a PhysBone where the mesh that exists as the target of the shaking is always disabled, an extra computational load is incurred.

Reviewer: ```suggestion
  - If there are enabled PhysBones which do not need to be swayed (e.g., the swaying target mesh is always disabled), extra computational load is caused.
```
ついでにanが不要そうだったので取り除きました

> - Merging bones being animated nor swayed with PhysBones

Author: 根拠がなくてコメントが遅くなりました。多分これはnot A or Bにすべきでnorジャなさそうですね。
```suggestion
- Merging bones which is not animated or swayed with PhysBones
```

> - Merging bones being animated nor swayed with PhysBones

Reviewer: 前回norが使われていて、軽く調べて問題があると判断出来なかったのがあったのでそのままにしてました
https://github.com/anatawa12/AvatarOptimizer/blob/a65e4bb24da065893cdb64dc2d99ab03abdaf074/CONTRIBUTING.md?plain=1#L31

  - When clothing bones are nested inside the bones of the body, there will be many bones that do not move on their own. Such bones create extra load.

> With Avatar Optimizer, You can those optimization with adding `Trace And Optimize` to the Avatar Root!

Reviewer: ```suggestion
With AvatarOptimizer, these optimizations can be performed automatically by simply adding a `Trace And Optimize` to the Avatar Root!
```

よく見ると英語版説明の他の箇所では"component"表記を使っていないっぽい?

> With Avatar Optimizer, You can those optimization with adding `Trace And Optimize` to the Avatar Root!

Author: 多分なしで書いてますね。明確な理由は多分ないです

> With Avatar Optimizer, You can those optimization with adding `Trace And Optimize` to the Avatar Root!

Reviewer: "component"表記の追加は気が向いたらPRにします

![add-trace-and-optimize.png](add-trace-and-optimize.png)

[^blend-shape]: BlendShapeはUnity上のシェイプキーの名前です。UnityやMayaではBlendShape、BlenderではShape Key、MetasequoiaやMMDではモーフと呼ばれます。

Merge Meshes to reduce # of Skinned Renderers {#merge-skinned-mesh}
--

You can easily merge Skinned Mesh with Avatar Optimizer!

> Merging Skinned Mesh will not allow you to turn them on and off individually, but combining them will save some rendering weight!

Author: たしかにcostのほうが良さそうですね

[^merge-skinned-mesh]: Root Bone and Anchor Override are impossible to merge automatically I think. If you know any good algorithm, please tel me that.
[^mesh]: In this document mesh means SkinnedMeshRenderer, not the Mesh asset in Unity.

> Reduce polygons with shrinking which shrinks parts of body

Author: TODO: Anchor置く

---

By deleting polygons that are hidden by clothing or otherwise, you can reduce rendering load, BlendShape processing load, etc., without affecting the appearance much.

> To easily achieve this, AvatarOptimizer can remove meshes using the BlendShapes which shrinks parts of body included in many avatars!

Reviewer: ```suggestion
To make this easier, AvatarOptimizer allows you to remove polygons with BlendShapes which many avatars have for shrinking parts of the body!
```

Author: `for shrinking parts of the body`の部分は制限的にしたくて、`many avatars have`は非制限的だと思うので、
`BlendShapes for shrinking parts of the body, which many avatars have`とかのほうがいいかも

Author: ```suggestion
To make this easier, AvatarOptimizer allows you to remove polygons with BlendShapes for shrinking parts of the body, which many avatars have!
```

> Let's add `Remove Mesh By BlendShape` to Body Mesh!

Reviewer: 前述の理由から、componentを付けるか否か

> Enable `Automatically set BlendShape weight for preview when toggled` to make sure that unexpected parts of the body are not removed,

Reviewer: ```suggestion
Enable `Automatically set BlendShape weight for preview when toggled` to make sure that unintended parts of the body are not removed, and
```

> Select the BlendShapes which shrinks parts of body which you want to remove from the list of BlendShapes below!

Reviewer: ```suggestion
select the BlendShapes, which shrink the parts of the body you want to remove, from the BlendShapes list below!
```

Author: 関係代名詞の制限用法なので,は入れてはいけない認識です

Reviewer: 選びうるBlendShapeにはshrinkしないものもあり得るということでしょうか

Reviewer: いや、theがある程度ではBlendShapeに掛かる関係代名詞として重いということか…

Author: 「BlendShapeのうち、消したい部分をシュリンクしてるの」という名詞句なので制限用法だと思う

Author: というかtheはないほうが良さそうかも

Reviewer: ```suggestion
select the BlendShapes from the BlendShapes list below that shrink the parts of the body you want to remove!
```
前後を入れ替えてthatにしてみました
正直自信は無いですが、少なくとも元のままだと、fromがどこに掛かるのかなど色々読みづらいと思っています

Reviewer: > というかtheはないほうが良さそうかも

反映しました
